client/state/sites/reducer.js

```diff
@@ -103,8 +103,6 @@ export function items( state = {}, action ) {
 			}, initialNextState );
```

Comment:
Thanks for chiming in @aduth.

So yeah, the thing is that in our particular case here with Jetpack Connect a site that we're not tracking CAN appear after we've started the request to fetch all sites. So we'll have to keep this check for good, and this only confirms that we'll have that ditch comment from there.